Sources/Sentry/Public/Sentry.h

```diff
@@ -1,46 +1,49 @@
-#import <Foundation/Foundation.h>
+#if !__has_include(<SentryWithoutUIKit/Sentry.h>)
```

Comment:
why this and not `__has_include(<Sentry/Sentry.h>)`?

Reply:
No specific reason. Yours is nicer because it avoids the double negative 😄 I'll change it and test, and if all looks good, we'll get this merged.

Reply:
Everything seems to work 👍🏻